tests/stdlib/tosenv.nim

```diff
@@ -0,0 +1,39 @@
+discard """
+  matrix: "--threads"
+  joinable: false
+  targets: "c cpp"
+"""
+import os
+
```

Comment:
can be done in future work:

next step, how about follow this pattern: https://github.com/timotheecour/Nim/issues/534#issuecomment-769565033
more especifically
* adding: js to the targets (it almost works except for a few cases which can be disabled by `when not defined(js): ...`)
* testing VM by wrapping all code in `template main() =... ; static: main(); main()` as shown in that link

note, this could wait for https://github.com/nim-lang/Nim/pull/18568 so that delEnv actually works in VM instead of being a noop (which i just discovered while trying this)

Reply:
👍🏻 I'll add it to future work above

Reply:
sure; let's leave this comment open then :)

Reply:
fixed in https://github.com/nim-lang/Nim/pull/18575